bakerydemo/templates/breads/bread_page.html

```diff
@@ -10,7 +10,7 @@
                     <h1>{{ page.title }}</h1>
                     
                     <figure class="hidden-md-up">
-                        {% image self.image width-600 %}
+                        {% image self.image width-500 %}
```

Comment:
This means we only need to load a single image asset for the mobile viewport and desktop, which makes me feel a little better about using the visibility class.